src/base/ApplicationLoadBalancer.spec.ts

```diff
@@ -29,4 +29,104 @@ describe('ApplicationLoadBalancer', () => {
     });
     expect(synthed).toMatchSnapshot();
   });
+
+  it('renders an ALB with logs with a new bucket', () => {
+    const synthed = Testing.synthScope((stack) => {
+      new ApplicationLoadBalancer(stack, 'testALB', {
+        prefix: 'test-',
+        alb6CharacterPrefix: 'TEST',
+        vpcId: '123',
+        subnetIds: ['a', 'b'],
+        tags: {
+          name: 'thedude',
+          hobby: 'bowling',
+        },
+        accessLogs: {
+          bucket: 'logging-bucket',
+        },
+      });
+    });
+    expect(synthed).toMatchSnapshot();
+  });
+
+  it('renders an ALB with logs with a new bucket and prefix', () => {
+    const synthed = Testing.synthScope((stack) => {
+      new ApplicationLoadBalancer(stack, 'testALB', {
+        prefix: 'test-',
+        alb6CharacterPrefix: 'TEST',
+        vpcId: '123',
+        subnetIds: ['a', 'b'],
+        tags: {
+          name: 'thedude',
+          hobby: 'bowling',
+        },
+        accessLogs: {
+          bucket: 'logging-bucket',
+          prefix: 'logs/ahoy/cool/service',
```

Comment:
calling out none of these `prefix` values have a trailing slash 🙈

Reply:
lol you are right. imma throw an error.